### PR TITLE
baseCounterName

### PR DESCRIPTION
**Pull request by FostUK**
Adds baseCounterName to example output and update docs for gmi.

### CODE REVIEW

File: docs/gmi.md [hunk 1]
```diff
@@ -47,11 +47,14 @@ e.g. console.log.
 
 ## Stats
 
-The sending of stats is now handled by GMI. A simple call can be used:
+The sending of stats is handled by GMI. 2 calls can are needed:
```
Comment (Leonie-) [inline docs/gmi.md]: Think this needs a bit of rewording: "2 calls can are needed"

Comment (Asa-Dawson) [inline docs/gmi.md]: Probably want to redact "can"

File: src/main.js [hunk 2]
```diff
@@ -96,8 +96,8 @@ define(["storage", "websockets", "account/morph-props"], function(storage, ws, P
     appendParagraph(" Click the \"Log setStatsEvent\" button to fire setStatsEvent.");
```
Comment (Leonie-) [inline src/main.js]: Is it worth mentioning that you need to open the dev tools to see the console.log output when you've clicked the button?

File: docs/gmi.md (hunk 1 above)
Comment (FostUK) [reply docs/gmi.md]: Yeah that made no sense :) Fixed now.

File: src/main.js (hunk 2 above)
Comment (FostUK) [reply src/main.js]: Good point. Added.

**Pull request merged**